### ISSUE TITLE
`memeApp.ipa` resource file

### ISSUE THREAD

**Issue by razvand** [klassiker/ctf-writeups#2]
I'm using [your write-up for the Great Success task](https://github.com/klassiker/ctf-writeups/blob/master/2020/appsec-il/greatsuccess.md) as part of a security class. But I wasn't able to locate the `memeApp.ipa` resource file. Do you still have access to it?

**Comment by klassiker** [comment]
No, I'm sorry. I've never added any source files to this repository and regularly cleaned my CTF directory. You could try asking the original challenge authors.

**Comment by razvand** [comment]
OK, thanks, I will ask them.
